server/handlers/handlers.go

```diff
@@ -25,11 +26,23 @@ const (
 	numberOfWorkers       = 20
 	activityQueueSize     = 1000
 	msteamsUserTypeGuest  = "Guest"
+
+	discardedReasonNone                   = ""
+	discardedReasonInvalidChangeType      = "invalid_change_type"
+	discardedReasonIsBotUser              = "is_bot_user"
+	discardedReasonUnableToGetTeamsData   = "unable_to_get_teams_data"
+	discardedReasonNotUserEvent           = "no_user_event"
+	discardedReasonOther                  = "other"
```

Comment:
I'm using this other to not put every single possible type of failure here. But this are counters, so I don't think is that problematic, we only process 3 change types, so the metrics are going to be 3 * number of errors, and because it is a counter, it only add item to the list per combination.